client/blocks/conversation-caterpillar/index.jsx

```diff
@@ -0,0 +1,84 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { map, get, last } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/***
+ * Internal dependencies
+ */
+import Gravatar from 'components/gravatar';
+import { getPostCommentsTree } from 'state/comments/selectors';
+
+class ConversationCaterpillarComponent extends React.Component {
+	static propTypes = {
+		blogId: PropTypes.number.isRequired,
+		postId: PropTypes.number.isRequired,
+	};
+
+	render() {
+		const { commentsTree, translate } = this.props;
+		const lastComment = commentsTree[ last( Object.keys( commentsTree ) ) ];
```

Comment:
Updated to use the selector's array of date-sorted comments rather than the `commentsTree`.